src/plugins/data/public/index.ts

```diff
@@ -170,7 +169,7 @@ import {
   TruncateFormat,
 } from '../common/field_formats';
 
-import { DateFormat } from './field_formats';
+import { DateNanosFormat, DateFormat } from './field_formats';
```

Comment:
There wasn't intended to be any impact to existing `DateNanosFormat` functionality, but somehow it looks like this PR made this format unavailable in the browser.

Error captured while viewing an index pattern that has a `date_nanos` formatted field:
![image](https://user-images.githubusercontent.com/908371/86974465-3d095e00-c12b-11ea-8047-2bc531634b02.png)